### ISSUE TITLE
Firefox/Chrome/Web browser add-on

### ISSUE THREAD

**Issue by Maijin** [MISP/MISP#958]
Related to #957. It could cool to have simple add-on that once linked to MISP instance you click on a button and it highlights the IOCs available on the page if present in the MISP instance + mouse overlay to get info. This may particularly interesting for example when you browse website such as urlquery or reports page where you have domains, hashes display.

**Comment by iglocska** [comment]
Very cool idea!

**Comment by advancedeng** [comment]
Definitely. Something like what iSight Partners provide would be more than ideal. :)
https://chrome.google.com/webstore/detail/threatscape-extension/ajdhfcoppbpocmaeoechanfllkoemlhi

Cheers,
Andreas

**Comment by jaegeral** [comment]
https://github.com/cloudtracer/ThreatPinchLookup looks like that is something.

(even if it does not work for me right now out of the box it looks very good)
